requirements/requirements.txt

```diff
@@ -1,4 +1 @@
-git+https://github.com/facebookresearch/hydra#subdirectory=tools/configen
-git+https://github.com/facebookresearch/hydra
-torch==1.6.0
-torchvision==0.7.0
+hydra-core==1.1.0.dev5
```

Comment:
no dependency on configen itself?

Reply:
> I feel uncomfortable including those huge test binaries in the repo.

hmm ok let me see what I can do about that

Reply:
Don't you think configen should only be a dev dependency? Users of these packages don't need to have `configen` installed. I'll add it to `reqs/dev.txt`. Do you think we should pin that or track a version? It's a little trickier since its still changing often.

Reply:
ah, yes - absolutely a dev dependency.

Reply:
Just to be more explicit, I'm opening a diff PR for this.

Reply:
actually, it's already in the `requirements/dev.txt` on `master`. I still need to open a new PR to change the structure for the configen >= dev8 though